atomic_reactor/core.py

```diff
@@ -620,3 +621,32 @@ def get_version(self):
         :return: dict, json output of `docker version`
         """
         return self.d.version()
+
+    def get_volumes_for_container(self, container_id):
+        """
+        get a list of volumes mounter in a container
+
+        :param container_id: str
+        :return: list, a list of volume ids
+        """
+        logger.info("listing volumes for container '%s'", container_id)
+        inspect_output = self.d.inspect_container(container_id)
+        volumes = inspect_output['Mounts'] or []
+
+        volume_names = [x['Name'] for x in volumes if x['Source'] != ""]
+        logger.debug("volumes = '%s'", volume_names)
+        return volume_names
+
+    def remove_volume(self, volume_id):
+        """
+        remove a volume by its id
```

Comment:
Agreed, I've renamed all 'id' to 'name', as volumes are identified by names anyway